packages/ethers-cf-purelocal/tsconfig.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
You can inherit from [`tsconfig.json`](https://github.com/counterfactual/monorepo/blob/master/tsconfig.json)